main.tf

```diff
@@ -8,14 +8,15 @@ resource "google_container_cluster" "this" {
   name     = var.GKE_CLUSTER_NAME
   location = var.GOOGLE_REGION
 
-  initial_node_count       = 1
+  initial_node_count       = 3
```

Comment:
this is a pool that will be deleted - do we want 3 nodes here?

Reply:
I decided to make changes (easy way) to increase the number of clusters and the type of machine, and then can destroy everything "terraform destroy"

Reply:
this is a default pool. It does nothing but just will be deleted.